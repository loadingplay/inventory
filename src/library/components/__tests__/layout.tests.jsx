import React from "react";
import { shallow } from 'enzyme';

import Layout from "../layout";

describe("Orders", () => {

  beforeEach(jest.clearAllMocks);

  test("it should asign right access token", () => {
    const wrapper = shallow(
      <Layout >test</Layout>,
      {
        disableLifecycleMethods: true
      }
    );
    expect(wrapper).toMatchSnapshot("default");
    expect(wrapper.children().debug()).toEqual("test");
  });
});
