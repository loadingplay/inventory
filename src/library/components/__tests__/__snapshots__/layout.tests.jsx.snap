// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Orders it should asign right access token: default 1`] = `
<div
  className="login_wrapper"
>
  test
</div>
`;
